app/javascript/hooks/queries/site_settings/useSiteSettingAsync.jsx

```diff
@@ -6,7 +6,7 @@ export default function useSiteSettingAsync(name) {
     ['getSiteSettings', name],
     async () => {
       const promise = await axios.get(`/site_settings/${name}.json`);
-      return promise.data;
```

Comment:
This mutation is duplicated in favor of `useSiteSetting`.
`useSiteSetting` is used in `FeatureTabs`.
`useSiteSettingAsync` was introduced because of this issue, because it's fixed `useSiteSetting` should be used **only**.

Reply:
I'll switch it in a separate PR

Reply:
Alright,